lib/models/addon.js

```diff
@@ -1445,4 +1442,112 @@ Addon.lookup = function(addon) {
  - [ember-cli-google-analytics](https://github.com/pgrippi/ember-cli-google-analytics/blob/v1.5.0/index.js#L79)
 */
 
+function methodsForTreeType(treeType) {
+  var treeMethods = DEFAULT_TREE_FOR_METHOD_METHODS[treeType];
+
+  return GLOBAL_TREE_FOR_METHOD_METHODS.concat(treeMethods);
+}
+
+if (experiments.ADDON_TREE_CACHING) {
+  addonProto[experiments.ADDON_TREE_CACHING] = function cacheKeyForTree(treeType) {
```

Comment:
`if (experiments.ADDON_TREE_CACHING) {` => implies boolean
`addonProto[experiments.ADDON_TREE_CACHING] =` => implies key

These feel very weird adjacent to each other. Can we at least `!!experiments.ADDON_TREE_CACHING` to make it a bit more clear?